app/src/main/java/ch/epfl/sdp/drone3d/ui/mission/ItineraryShowActivity.kt

```diff
@@ -50,6 +58,10 @@ class ItineraryShowActivity : BaseMapActivity() {
 
     private lateinit var deleteButton: MaterialButton
 
+    // true if the weather is good enough to launch the mission
+    private var isWeatherGoodEnough: Boolean = false
```

Comment:
Why not making isWeatherGoodEnough also lateinit?

Reply:
isWeatherGoodEnough can't be lateinit as it is a boolean and lateinit don't work on primitive types